librz/core/cmd_descs/cmd_debug.yaml

```
@@ -16,6 +16,7 @@ commands:
         type: RZ_CMD_DESC_TYPE_ARGV_STATE
         modes:
            - RZ_OUTPUT_MODE_STANDARD
+           - RZ_OUTPUT_MODE_TABLE
```

```suggestion
        default_mode: RZ_OUTPUT_MODE_TABLE
        modes:
           - RZ_OUTPUT_MODE_TABLE
```

FYI, this is also an option if we want. It makes the default mode (the one you get when just doing `dbl`) the table mode.

---

Oh, didn't know that. Thanks. But I don't think we should remove the standard mode output. What if the user wants to use standard mode output?

---

When you use default_mode there is no STANDARD mode anymore. The standard mode becomes the TABLE. One could argue what is the additional benefit of a "STANDARD" mode vs the table mode, which has the same info, with headers, with a syntax to filter columns/rows/etc.

---

Oh, yes. I was talking about the plain standard mode only. I thought that is more concise and has less fluff, but I also agree that it is not especially useful compared to the other modes. I'll do this.